package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "airswap.js",
-  "version": "0.1.112",
+  "version": "0.1.113",
```

Comment:
the newest version on NPM is 0.1.112

https://www.npmjs.com/package/airswap.js